spacy/pipeline/entityruler.py

```diff
@@ -276,7 +279,7 @@ def add_patterns(self, patterns: List[PatternType]) -> None:
                     current_index = i
                     break
             subsequent_pipes = [
-                pipe for pipe in self.nlp.pipe_names[current_index + 1 :]
+                pipe for pipe in self.nlp.pipe_names[current_index :]
```

Comment:
@svlandeg: Is there a good reason to run the entity ruler itself as part of the pipeline when adding phrase patterns? It seems a little circular to me, but maybe there's a reason I'm missing?

Reply:
Huh, no idea. This seems to be in the code base already for some time. I think your edit should be fine?

Reply:
It seems okay in all my tests, so I'll leave it for now.